snowdrop-core/src/Snowdrop/Core/ERoComp/Helpers.hs

```diff
@@ -72,21 +72,28 @@ data TxValidationException
 
 instance Buildable TxValidationException where
     build = \case
-        ProofProjectionFailed tx -> bprint ("Projection of proof is failed during validation of StateTx with type: "%build) tx
+        ProofProjectionFailed tx ->
+            bprint
+              ("Projection of proof is failed during validation of StateTx with type: "%build)
+              tx
         PayloadProjectionFailed tx p ->
-            bprint ("Projection of payload is failed during validation of StateTx with type: "%build%", got prefix: "%build) tx p
+            bprint
+              ("Projection of payload is failed during validation of StateTx with type: "
+               %build%", got prefix: "%build)
+              tx
+              p
         UnexpectedPayload p -> bprint ("Unexpected payload, prefixes: "%listF ", " build) p
 
 -- | For each id' from passed set of ids get value' from state of computation.
 -- HasAlt id id' satisfies that @id@ may be injected from @id'@.
 -- If some value' couldn't be projected from corresponding value, the request will fail.
 querySet
-  :: forall id id' value value' e ctx .
-     ( Ord id, Ord id'
-     , HasKeyValue id value id' value'
-     , HasException e StatePException
-     )
-  => Set id' -> ERoComp e id value ctx (StateP id' value')
+    :: forall id id' value value' e ctx .
```

Comment:
it seems you aren't consistent with yourself: in one places `::` and `(` in one column, in others in different